content/02.body.md

```diff
@@ -232,5 +230,18 @@ Second, node pair probabilities can be used to adjust edge prediction features d
 If degree is a desired feature, then the edge prior can be treated like a Bayesian prior probability.
 Alternatively, if degree is not a desired feature, then the edge prior can be used to calibrate features and thus potentially enhance predictive specificity.
 
+Figure {@fig:feature-auroc} illustrates the utility of the edge prior and permutation framework for two purposes.
+First, it contextualizes feature performances relative to the baseline of nonspecific, degree-based predictions, quantified by the edge prior.
+Degree has varying utility for different edge prediction tasks.
+The edge prior's performance on a task quantifies the utility of degree toward the task.
+This comparison is useful because specific predictions (based on more than degree alone) are more valuable for some applications than nonspecific ones and because degree can be an expression of bias in many real-world networks.
+
+Second, Figure {@fig:feature-auroc} visualizes a comparison of five edge prediction features computed on and unpermuted networks.
+This comparison identified the fraction of each feature's performance attributable to degree.
+Some features, such as the preferential attachment index, pick up perfectly and exclusively on degree.
+In the comparison, this feature had identical performance regardless of the train network.
+Conversely, we can see that the Adamic/Adar index picked up almost perfectly on degree because its performances from permuted networks are nearly at the performance of the edge prior.
+Moreover, the Adamic/Adar index had much higher performance when computed on the unpermuted network, indicating that it also picks up on higher-order information in addition to degree.
```

Comment:
I think what you want to say is: "this other thing got degree, but also other stuff n' things.